### ISSUE TITLE
Obs dim mismatch: xArm6 and Panda

### ISSUE THREAD

**Issue by spacegoing** [ExistentialRobotics/cross_embodiment_transfer#2]
How to reproduce the xArm6 experiments with simulated environments?

I notice that xArm6 has 6+6=12 obs dim while Panda / Sawyer has 7+7=14 obs dim whie generating demos.

How to align this dimension during alignment stage?

Many thanks!

**Comment by szlww** [comment]
@spacegoing Hi, sorry for border you. Have you reproduce the Sawyer experiment with simulated environment? I reproduced it but the result of alignment was bad, did you meet this problem?

**Comment by spacegoing** [comment]
@szlww 
unfortunately I wasn't able to run it

**Comment by Shaocheng-Liu** [comment]
@spacegoing Hi, I also met this problem when I try to rerun this code, and I assume that the last part of the align.py should be changed.
```python
            # real_tgt_obs = self.src_obs_dec(real_lat_obs)
            real_tgt_obs = tgt_obs
            # real_tgt_act = self.src_act_dec(torch.cat([real_tgt_obs, real_lat_act], dim=-1))
            real_tgt_act = tgt_act
            fake_tgt_obs = self.tgt_obs_dec(real_lat_obs)
            fake_tgt_act = self.tgt_act_dec(torch.cat([fake_tgt_obs, real_lat_act], dim=-1))
            # fake_tgt_act_1 = self.tgt_act_dec(torch.cat([real_tgt_obs, real_lat_act], dim=-1))
            L.add_scalar('valid_lat_gen/tgt_obs_diff', F.l1_loss(real_tgt_obs, fake_tgt_obs), step)
            L.add_scalar('valid_lat_gen/tgt_act_diff', F.l1_loss(real_tgt_act, fake_tgt_act), step)
            # L.add_scalar('valid_lat_gen/tgt_act_diff', F.l1_loss(real_tgt_act, fake_tgt_act_1), step)
```
I'm not sure if it is correct but at least there is no error for xArm6, maybe we could have some discussion about it?
